curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f5c3e399ff1a05629964e4.md

```diff
@@ -30,6 +30,12 @@ You should give the `action` a value of `https://register-demo.freecodecamp.org`
 assert.equal(document.querySelector('form')?.action, 'https://register-demo.freecodecamp.org/');
 ```
 
+Your `form` element should have a closing tag. Closing tags have a `/` just after the `<` character.
```

Comment:
```suggestion
Your `form` element should have a closing tag `</form>`.
```

I know I have seen this hint for closing tags elsewhere, but in my opinion, it is a bit ambiguous, and potentially confusing.